### PR TITLE
Network permissioning new doc

### PR DESCRIPTION
**Pull request by patrickkuo**

### CODE REVIEW

File: docs/source/permissioning.rst [hunk 1]
```diff
@@ -1,104 +1,58 @@
...
-    violates legal name rules, see `Legal Name Constraints`_ for more information.
+Create the certificate authority
+--------------------------------
 
-:emailAddress: e.g. "admin@company.com"
+* Create root CAs and truststore
+1, Create a new keypair
+2, Self sign the certificate, with basic constraints extension set to ``true``, this will be used asthe root CA keys and root CA certificate.
+3, Store the root keypair and the certificate to a keystore for later use.
+4, Store the root CA's certificate with to a java keystore with alias "cordarootca" and name the keystore "truststore.jks",
```
Comment (gendal) [inline docs/source/permissioning.rst]: delete the first "with"?

File: docs/source/permissioning.rst [hunk 2]
```diff
@@ -1,104 +1,58 @@
...
+this will need to be distributed to every node.
```
Comment (gendal) [inline docs/source/permissioning.rst]: Period then capital letter for "This".

File: docs/source/permissioning.rst [hunk 3]
```diff
@@ -1,104 +1,58 @@
...
-
-* Should start with a capital letter.
-
-* No commas or equals signs.
-
-* No dollars or quote marks, although we may relax the quote mark constraint in future to handle Irish company names.
-
-Starting the Registration
--------------------------
+.. note:: We call this "Node CA" because it has the authority to issue child certificates, this is used to sign identity keys and anonymous keys
```
Comment (gendal) [inline docs/source/permissioning.rst]: Period, then new sentence between "certificates" and "this".

File: docs/source/permissioning.rst [hunk 4]
```diff
@@ -1,104 +1,58 @@
...
+compromising the root private key.
 
-.. note:: You can exit the utility at any time if the approval process is taking longer than expected. The request process will resume on restart. 
+Create node identity keystore and TLS keystore
+----------------------------------------------
 
-This process only is needed when the node connects to the network for the first time, or when the certificate expires.
+* Create Node CA and TLS keystore
+1, On each node, create a new keypair.
+2, Sign the certificate using the intermediate CA key with basic constraints extension set to ``true``
```
Comment (gendal) [inline docs/source/permissioning.rst]: What certificate?  Do we need to explain what the cert needs to look like?  Or tell people how to get the node to create a CSR on the file system? (with a PR to make this possible if it isn't already?).

**Comment by joeldudleyr3** [comment]
@gendal Going through this now. I'll push up a commit in a sec covering the typos you identified and a few other thing ( @patrickkuo - hope you don't mind :) )

**Comment by joeldudleyr3** [comment]
@patrickkuo I've finished my rewording. Can you check it's accurate?

There's also a few more things missing IMO:
- Can you provide a quick description of the purpose of each of the three keystores at the top?
- I think we should document which keypair formats are supported - this isn't anywhere else in the docs
- Just to be clear - if she support any certificate format, we should change this to be explicit on that point

**Pull request merged**